src/app/data/assets/metadata.json

```diff
@@ -318,6 +318,17 @@
             "</p>"
         ]
     },
+    {
+        "subvariant": "cosmic",
+        "category": "spins",
+        "name": "Fedora COSMIC Desktop",
+        "summary": "Fedora COSMIC Desktop",
+        "description": [
+            "<p>",
+            "Fedora COSMIC Desktop",
+            "</p>"
```

Comment:
Can we have the description synced from [the page](https://fedoraproject.org/spins/cosmic)?

Reply:
The description is not used, it's just a placeholder basically. It's a relief from the past when we had the description for each variant. The only reason it's there is because nobody has touched the model that expects this field to be set.

Reply:
Ah okay.